src/index.js

```diff
@@ -134,6 +149,13 @@ class Anvil {
     }
   }
 
+  downloadDocuments (documentGroupEid) {
+    return this.requestREST(
+      `/api/document-group/${documentGroupEid}.zip`,
+      { method: 'GET' },
+      { dataType: DATA_TYPE_STREAM })
```

Comment:
Why default to Stream here vs Buffer (which is the default for `fillPDF`? And why not allow it to be overridden to be the other type (Buffer in this case)?

Reply:
Defaulted to stream since it was how I used the API in the example repo. But I've just added the option to choose between `stream` and `buffer`, following the approach of `fillPDF` with `buffer` being the default.